README.md

```diff
@@ -186,13 +186,15 @@ A `DIDAttributeChanged` event for the identity `0xf3beac30c498d9e26865f34fcaa57d
 The resolver presents a simple `resolver()` function that returns a ES6 Promise returning the DID document.
 
 ```js
-import resolve from 'did-resolver'
-import registerResolver from 'ethr-did-resolver'
+import { Resolver } from 'did-resolver'
+import getResolver from 'ethr-did-resolver'
 
-registerResolver()
+const providerConfig = { provider, registry: registry.address }
 
-resolve('did:ethr:0xf3beac30c498d9e26865f34fcaa57dbb935b0d74').then(doc => console.log)
+const ethrDidResolver = getResolver()
+const didResolver = Resolver({ethr: ethrDidResolver})
```

Comment:
The `ethrDidResolver` MUST be assigned to the `ethr` key, otherwise it resolve correctly.  This is a potential source of error for devs to provide an incorrect configuration.  The resolver being registered should be responsible for setting the correct key following the existing pattern.

Reply:
I think it's best if `getResolver()` returns a mapping instead of the direct method.
Ex: `{ "ethr": resolve }`

This way it takes care of properly naming the method(s) and can also bubble up embedded resolvers.
Ex:
```js
{
    "ethr:mainnet" : resolve,
    "ethr" : resolve,
    "ethr:0x1" : resolve
}
```